api/USERS_GUIDE.md

```diff
@@ -109,6 +109,11 @@ available endpoints and their parameters and return values.
 
 ### Project searches
 
+Use URL parameters to search for projects against the `v1/projects` endpoint.
+If different parameters are included they are ANDed together. To send multiple
+values of the same parameter to OR together, append the name with `[]`
+(e.g. `projectid[]=`).
```

Comment:
we may want to clarify that the unique parameters are still ANDed, just OR among parameters of the same type.

Reply:
I've rephrased it -- is that clearer?

Reply:
Much better in my opinion. Thank you